vtex/admin-ui
# Build/add rollup

#### What is the purpose of this pull request?
Create a new build config using Rollup + Turborepo.

#### What problem is this solving?
We had some problems with typings in the last release, blocking it to be used until release a fixed version. That issue Indicated that we have room for improvement in our admin-ui build process.

#### How should this be manually tested?
Run `yarn build` and check the build files in the `dist` folder for each package.

**Experimental version built with the new config**
https://bundlephobia.com/package/@vtex/admin-ui@0.133.7-experimental.11

![Screen Shot 2022-09-08 at 15 34 58](https://user-images.githubusercontent.com/991309/189199507-0be49cb6-62ad-438f-aead-e34163b619d4.png)


#### Types of changes
